### PR TITLE
[WIP] cleanup, remove old 0.4 code

### PR DESCRIPTION
**Pull request by Niklp09**
WIP

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -53,56 +53,20 @@
...
-		end
-		if not canExist then
-			self.object:remove()
-		end
-	end
-})]]
 
-if minetest.global_exists("areas") then
+if debug then -- hud id for debug data
+	hangglider.debug = {}
```
Comment (BuckarooBanzay) [inline init.lua]: luacheck complains here about trailing space:
```
    init.lua:66:23: line contains trailing whitespace
```

**Review by BuckarooBanzay — APPROVED** [review]
other than the one luacheck issue: lgtm 👍

**Comment by Niklp09** [comment]
This PR is not ready yet :P

File: init.lua [hunk 2]
```diff
@@ -115,8 +79,7 @@ if minetest.global_exists("areas") then
 			end
 
 			if not areas:isAreaOwner(id, name) then
-				return false, "Area "..id.." does not exist"
-					.." or is not owned by you."
+				return false, "Area "..id.." does not exist".." or is not owned by you."
```
Comment (S-S-X) [inline init.lua]:
```suggestion
				return false, "Area "..id.." does not exist or is not owned by you."
```

File: init.lua [hunk 3]
```diff
@@ -268,73 +229,53 @@ minetest.register_on_joinplayer(function(player)
...
-				end, stopper, player)
-			end]]
 			if not airbreak then
-				if moveModelUp then
-					minetest.add_entity(pos, "hangglider:glider"):set_attach(player, "", {x=0,y=10,z=0}, {x=0,y=0,z=0})
-				else
-					minetest.add_entity(pos, "hangglider:glider"):set_attach(player, "", {x=0,y=0,z=0}, {x=0,y=0,z=0})
-				end
+				minetest.add_entity(pos, "hangglider:glider"):set_attach(player, "", {x=0,y=10,z=0}, {x=0,y=0,z=0})
 			end
```
Comment (S-S-X) [inline init.lua]: To cleanup bit more just replace this whole block with single line:
```
minetest.add_entity(pos, "hangglider:glider"):set_attach(player, "", {x=0,y=10,z=0}, {x=0,y=0,z=0})
```

`airbreak` variable is never really used.

**Review by S-S-X — APPROVED** [review]
Two minor things / additional cleanup. Approving with or without those changes.

**Pull request merged**